sql/core/src/test/resources/sql-tests/results/postgreSQL/union.sql.out

```diff
@@ -682,10 +682,10 @@ struct<id:bigint>
 -- !query
 SELECT cast('3.4' as decimal(38, 18)) UNION SELECT 'foo'
 -- !query schema
-struct<CAST(3.4 AS DECIMAL(38,18)):string>
+struct<>
 -- !query output
-3.400000000000000000
-foo
+org.apache.spark.SparkException
```

Comment:
See the discussion in https://github.com/apache/spark/pull/31390#issuecomment-783116851.
The function `btrim` was added for string type as per @beliefer

Reply:
It seems nothing to do with `btrim`.

Reply:
oh, wrong place...